app/views/experiences/show.html.erb

```diff
@@ -99,8 +99,8 @@
     <div class='row'>
       <div class='col-md-10 col-md-offset-1'>
 
-        <h2 class='section-title text-submit'>Know people who have something to add to this conversation?</h2>
-        <p class='section-description'>More people means more problems solved. Share this challenge!</p>
+        <h2 class='section-title text-submit'><%= t('experiences.show.social_sharing_question') %></h2>
+        <p class='section-description'><%= t('experiences.show.social_sharing_description') %></p>
```

Comment:
There's about nine of these properties repeated throughout.  It'd be best if they were coalesced into one common property.